gpytorch/priors/wishart_prior.py

```diff
@@ -61,7 +61,7 @@ def __init__(self, nu, K):
             raise ValueError("Must have nu > 0")
         super(InverseWishartPrior, self).__init__()
         self.register_buffer("K", K)
-        self.register_buffer("nu", torch.Tensor([nu]))
+        self.register_buffer("nu", torch.tensor(float(nu), dtype=K.dtype))
```

Comment:
the `float` casting here is unnecessary if you already specify the `dtype` in the constructor

Reply:
Using `float(nu)` for consistency